core/src/main/java/org/mule/runtime/core/internal/context/notification/DefaultFlowCallStack.java

```diff
@@ -41,6 +50,16 @@ private DefaultFlowCallStack(final Deque<FlowStackElement> innerStack) {
    * @param flowStackElement the element to add
    */
   public void push(FlowStackElement flowStackElement) {
+    if (innerStack.size() >= MAX_DEPTH) {
+      StringBuilder messageBuilder = new StringBuilder();
+
+      messageBuilder.append("Too many child contexts nested. ")
```

Comment:
"Too many nested child context."

Reply:
![image](https://user-images.githubusercontent.com/8010105/69185972-7aff3680-0af6-11ea-9106-773fbf86f2ad.png)

Reply:
@elrodro83 please kindly stop using that image. I find it incredibly insensitive and not something I'd expect to see in a Mulesoft repo.